### PR TITLE
merged headerConfig object into appConfig.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/components/common/Header.jsx [hunk 1]
```diff
@@ -144,42 +144,46 @@ const Header = ({
...
-      </div>
-    );
-  } return (
-    <div style={routeType.default.headerWrapperStyle} className="student-logo-header print-media-none">
-      {RenderLogo(routeType.default)}
-      {RenderHeaderName(routeType.default)}
-      {RenderButton(routeType.default)}
-    </div>
-  );
+  return routes.map((rout) => {
```
Comment (mohitsaladiya) [inline src/components/common/Header.jsx]: typo
```suggestion
  return routes.map((route) => {
```

File: src/components/common/Header.jsx [hunk 2]
```diff
@@ -144,42 +144,46 @@ const Header = ({
...
+    if (rout.path === location) {
+      return (
+        <div key={rout.path} style={rout.header.headerWrapperStyle} className="student-logo-header print-media-none">
+          {RenderLogo(rout.header)}
```
Comment (mohitsaladiya) [inline src/components/common/Header.jsx]: Here as well
```suggestion
          {RenderLogo(route.header)}
```

File: src/components/common/Header.jsx [hunk 3]
```diff
@@ -144,42 +144,46 @@ const Header = ({
...
+          {RenderHeaderName(rout.header)}
+          {RenderButton(rout.header)}
+        </div>
+      );
+    } else if (location === '/files') {
+      return (
+        <div key={rout.path} style={rout.header.headerWrapperStyle} className="student-logo-header print-media-none">
+          {RenderLogo(rout.header)}
```
Comment (mohitsaladiya) [inline src/components/common/Header.jsx]: Same here
```suggestion
          {RenderLogo(route.header)}
```

File: src/components/common/Header.jsx [hunk 4]
```diff
@@ -144,42 +144,46 @@ const Header = ({
...
-    <div style={routeType.default.headerWrapperStyle} className="student-logo-header print-media-none">
-      {RenderLogo(routeType.default)}
-      {RenderHeaderName(routeType.default)}
-      {RenderButton(routeType.default)}
-    </div>
-  );
+  return routes.map((route) => {
+    if (route.path === location) {
+      return (
+        <div key={route.path} style={route.header.headerWrapperStyle} className="student-logo-header print-media-none">
```
Comment (pepcus-pratik) [inline src/components/common/Header.jsx]: Prefer object-destructuring for route.header

File: src/config/appConfig.json [hunk 5]
```diff
@@ -183,61 +184,129 @@
...
+      }
     },
     {
       "name": "adminPanel",
       "path": "/adminPanel",
       "component": "AdminPanel",
-      "isActive": false
+      "isActive": false,
+      "header": {
+        "HeaderContainedButton": true,
```
Comment (pepcus-pratik) [inline src/config/appConfig.json]:
```suggestion
        "hasButtons": true,
```

File: src/components/common/Header.jsx [hunk 6]
```diff
@@ -144,42 +144,46 @@ const Header = ({
...
+          {RenderLogo(route.header)}
+          {RenderHeaderName(route.header)}
+          {RenderButton(route.header)}
+        </div>
+      );
+    } else if (location === '/files') {
+      return (
+        <div key={route.path} style={route.header.headerWrapperStyle} className="student-logo-header print-media-none">
+          {RenderLogo(route.header)}
```
Comment (pepcus-pratik) [inline src/components/common/Header.jsx]: method names should be in lowerCamelCase

File: src/config/appConfig.json (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/config/appConfig.json]: Done.

File: src/components/common/Header.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

File: src/components/common/Header.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

File: src/components/common/Header.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

File: src/components/common/Header.jsx (hunk 6 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

File: src/components/common/Header.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

Comment (BhagyashreeBhoyar) [reply src/components/common/Header.jsx]: Done.

**Pull request merged**